packages/kotti-ui/source/kotti-modal/KtModal.vue

```diff
@@ -35,6 +35,18 @@ export default defineComponent<KottiModal.PropsInternal>({
 		const contentRef = ref<HTMLElement | null>(null)
 		const targetRef = ref<HTMLElement | null>(null)
 		const tippyInstanceRef = ref<Instance | null>(null)
+		/**
+		 * The `onCreate` function of `tippyjs` gets executed after the `KtModal`
+		 * component renders.
+		 *
+		 * This boolean that keeps track of the initial value of `props.isOpen`,
+		 * and passes it once tippy is created.
+		 *
+		 * It can be passed as a prop to tippy.
+		 * @see {@link https://atomiks.github.io/tippyjs/v6/all-props/#showoncreate}
+		 * However, for readability purposes, we manually show inside the `onCreate` function.
+		 */
+		const showOnCreate = ref(false)
```

Comment:
Can't you simply check the current value of `isOpen` and get rid of this variable?

Reply:
ye sure whatever

Reply:
on second thought, i prefer this because it's more readabile and doesn't leave room for future mistakes